build-scripts/gulp/clean.js

```diff
@@ -5,32 +5,32 @@ require("./translations");
 
 gulp.task(
   "clean",
-  gulp.parallel("clean-translations", function cleanOutputAndBuildDir() {
```

Comment:
The downside of this is that gulp won't be able to give the task a name.

Reply:
At least in the past.

Reply:
They actually advise not to use `gulp.task` anymore but to export a function... A job for another day...